packages/manager/apps/pci-ai-notebooks/src/components/order/framework/FrameworkSelect.component.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { Tabs, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import * as ai from '@/types/cloud/project/ai';
+import FrameworkTile from './FrameworkTile.component';
+import { FrameworkWithVersion } from '@/types/orderFunnel';
+
+interface FrameworkSelectProps {
+  frameworks: ai.capabilities.notebook.Framework[];
+  value: FrameworkWithVersion;
+  onChange: (newFrameworkWithVersion: FrameworkWithVersion) => void;
+}
+const FrameworksSelect = React.forwardRef<
+  HTMLInputElement,
+  FrameworkSelectProps
+>(({ frameworks, value, onChange }, ref) => {
+  const [selectedFrameworkTypeIndex, setSelectedFrameworkTypeIndex] = useState(
+    0,
+  );
+  const mappedFramework = frameworks.map((fmk) => ({
+    ...fmk,
+    type: fmk.docUrl ? 'classical' : 'quantique',
+  }));
```

Comment:
Yep, at terms it will come from the API. For the moment and for the test main point was to see how that can work with a tab component. It will be change once the API rework will have been done.